src/components/NetworkAssetSelectInput/NetworkAssetSelectInput.tsx

```diff
@@ -377,12 +436,21 @@ const NetworkAssetSelectInput = ({
         (item: any) => item.chain === supportedChain.chainId
       );
       return smartWalletBalanceByChain?.length && balanceByChain.length
-        ? formatAmountDisplay(String(balanceByChain[0].total), '$')
+        ? ` · ${formatAmountDisplay(String(balanceByChain[0].total), '$')}`
         : '$0';
-    } else if (accType === DestinationWalletEnum.Key && label === 'From') {
-      return '$0';
+    } else if (
```

Comment:
No need `else if` as previous condition returns the output so previous condition can be just closed and new put new `if`